amx/src/tmul.c

```diff
@@ -0,0 +1,968 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
@ysun 
Hi Yi Sun, I notice that Pengfei and Ammy all added declaration on the beginning in the *.c files. Do we have to align the need to add or not? If yes, I think it would be better if we follow the same format.

Reply:
I see.  Yes, that would be good if we follow the same format.
Let's note it in out To-do list. We can cook a patch to fix all that declaration format.